src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.DotNet.ILCompiler.targets

```diff
@@ -0,0 +1,85 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Isn't this file from the package? We can't move the targets out of the package as long as we still have consumers of the package (which is everyone right now). I think we have to start by integrating with the package into the SDK as-is for one release, then moving it over.

Reply:
I was thinking of copying the targets to the SDK and adding a guard to the SDK side and leave the targets in the package for backward compatibility. But your suggestion below sounds a reasonable one to take and move the targets to the SDK later when existing folks move.